plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/IDeployImagePageModel.java

```diff
@@ -94,7 +94,17 @@
 	 * @param imageName the name of the Docker Image to use
 	 */
 	void setImageName(String imageName);
-	
+
+	/**
+	 * Since method setImageName(String) ignores attempts to set an empty value, 
+	 * ui gets not synchronized with model as soon as image name is cleared.
```

Comment:
I dont fully get whey the image name resetl isnt propagated to the model and we're have to forcibly update? Where does the "name is cleared" happen? In the widget only? if so why is this not propagated to the model?